### PR TITLE
Remove auth

### PR DESCRIPTION
**Pull request by ungarson**

### CODE REVIEW

File: src/views/diff/Diff.vue [hunk 1]
```diff
@@ -1,168 +0,0 @@
...
-        <v-col cols="6" class="pr-1">
-          <v-skeleton-loader v-if="loading" type="card-heading, image"></v-skeleton-loader>
-        </v-col>
-        <v-col cols="6">
-          <v-skeleton-loader v-if="loading" type="card-heading, image"></v-skeleton-loader>
-        </v-col>
-      </v-row>
-      <v-row v-if="!loading && res" no-gutters>
-        <v-col cols="12">
-          <template v-if="isAuthorized">
```
Comment (m-kus) [inline src/views/diff/Diff.vue:15]: Only auth section should be removed, not the entire diff viewer

**Comment by ungarson** [comment]
[Remove views requiring auth](https://trello.com/c/gboWDq1U/1139-remove-views-requiring-auth)

File: src/views/diff/Diff.vue (hunk 1 above)
Comment (ungarson) [reply src/views/diff/Diff.vue:15]: @m-kus my bad, returned it back.

**Comment by ungarson** [comment]
@m-kus conflict is fixed & diff is back, ready to merge

**Pull request merged**